packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.Input/src/InputTypes/Serialization/TypeSpecInputEnumTypeConverter.cs

```diff
@@ -61,10 +61,6 @@ public static InputEnumType CreateEnumType(ref Utf8JsonReader reader, string? id
             }
 
             name = name ?? throw new JsonException("Enum must have name");
-            if (summary is null && doc is null)
-            {
-                Console.Error.WriteLine($"[Warn]: Enum '{name}' must have either a summary or doc");
```

Comment:
Did we intend to not replace this?

Reply:
I kind of think, it is find that in an unbranded library, some member does not have a doc, we do not really make a warning over it, right?

Reply:
we have this line because initially this was copied from autorest csharp which is designed for azure libraries and azure libraries require every member to have a doc.